frontend/src/exams/exam-inventory-table.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <div style="margin-left: 20px">
+    <b-form inline class="pb-2 pl-3">
+      <font-awesome-icon icon="filter"
+                         class="mr-1"
+                         style="font-size: 1.0rem"/>
+      <b-form-select :options="filters"
+                     @input.native="changeFilter"
+                     :value="filter"/>
+      <font-awesome-icon icon="binoculars"
+                         class="ml-3 mr-1"
+                         style="font-size: 1.0rem"/>
+      <b-form-input placeholder="type to search"/>
+    </b-form>
+    <b-table :items="examInventory"
+             :fields="fields"
+             class="m-0 p-0"
+             head-variant="light"
+             small
+             outlined
+             hover>
+      <template slot="roomloc" slot-scope="row">
+        <div v-if="row.item.location">
+          {{ row.item.location }}
+        </div>
+        <div v-if="row.item.room">
+          {{ row.item.room }}
+        </div>
+      </template>
+    </b-table>
+
+  </div>
+</template>
+
+<script>
+  import { mapGetters, mapState } from 'vuex'
+
+  export default {
+    name: "ExamInventoryTable",
+    props: ['mode'],
+    data() {
+      return {
+        filter: null,
+        filters: [
+          {text: 'Select a Filter', value: null},
+        ],
+        fieldsModal: [
+          {key: 'event_id', label: 'Event ID', sortable: false, thStyle: {width: '9%'} },
```

Comment:
Do we actually need width defined on every column?

Reply:
the default values it chooses are whack.  we specified a width for every column on the other tables.  i vote yes.